### PR TITLE
[HDInsight][Breaking Change]Remove deprecated parameters

### PR DESCRIPTION
**Pull request by aim-for-better**
## Description
Removed two parameters which have been marked as [deprecated] for a long time.

  - Removed the parameter `-RdpAccessExpiry` which has been marked as deprecated for a long time from cmdlet `New-AzHDInsightCluster`
  - Removed the parameter `-RdpCredential` which has been marked as deprecated for a long time from cmdlet `New-AzHDInsightCluster`

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by wyunchi-ms** [comment]
Hi @aim-for-better , your code include breaking changes. Please wait for next major release.

**Comment by aim-for-better** [comment]
Hi @wyunchi-ms @msJinLei  Could you please help review this PR so that it can catch the Breaking Change Review cycle?
Thank you very much.

**Comment by msJinLei** [comment]
@aim-for-better 
Please suppress breaking change errors
```
Release the breaking changes
Breaking changes can only be release in major release, usually twice a year. Major releases are identified in the [Release Schedule](https://github.com/Azure/azure-powershell/milestones).

Please commit your breaking changes to main in the sprint of major release.

In the commit, you should

suppress the breaking change errors in "tools/StaticAnalysis/Exceptions/${ModuleName}/BreakingChangeIssues.csv"
"AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
"Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzFirewallPolicyNatRuleCommand","New-AzFirewallPolicyNatRule","0","1050","The parameter set '__AllParameterSets' for cmdlet 'New-AzFirewallPolicyNatRule' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'New-AzFirewallPolicyNatRule'."

remove the unnecessary breaking change notifications
```

**Comment by aim-for-better** [comment]
Hi @msJinLei , Thanks for your comments.
I already added the breaking change suppression.  But there are still 4 pipelines failed. Could you please help on this? Thanks

File: src/HDInsight/HDInsight/help/New-AzHDInsightCluster.md [hunk 1]
```diff
@@ -802,7 +801,7 @@ The Premium tier can only be used with Linux clusters, and it enables the use of
 Type: System.String
 Parameter Sets: (All)
 Aliases:
-Accepted values: Standard, Premium
+-Accepted values: Standard, Premium
```
Comment (msJinLei) [inline src/HDInsight/HDInsight/help/New-AzHDInsightCluster.md:804]:
```suggestion
Accepted values: Standard, Premium
```
Remove ·-·， same below

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Comment by msJinLei** [comment]
> @aim-for-better Please suppress breaking change errors. The errors can be found in the static analysis result of azure-powershell - windows-powershell
> 
> ```
> Release the breaking changes
> Breaking changes can only be release in major release, usually twice a year. Major releases are identified in the [Release Schedule](https://github.com/Azure/azure-powershell/milestones).
> 
> Please commit your breaking changes to main in the sprint of major release.
> 
> In the commit, you should
> 
> suppress the breaking change errors in "tools/StaticAnalysis/Exceptions/${ModuleName}/BreakingChangeIssues.csv"
> "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
> "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzFirewallPolicyNatRuleCommand","New-AzFirewallPolicyNatRule","0","1050","The parameter set '__AllParameterSets' for cmdlet 'New-AzFirewallPolicyNatRule' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'New-AzFirewallPolicyNatRule'."
> 
> remove the unnecessary breaking change notifications
> ```

@aim-for-better Please refer to my comments

**Comment by aim-for-better** [comment]
> > @aim-for-better Please suppress breaking change errors. The errors can be found in the static analysis result of azure-powershell - windows-powershell
> > ```
> > Release the breaking changes
> > Breaking changes can only be release in major release, usually twice a year. Major releases are identified in the [Release Schedule](https://github.com/Azure/azure-powershell/milestones).
> > 
> > Please commit your breaking changes to main in the sprint of major release.
> > 
> > In the commit, you should
> > 
> > suppress the breaking change errors in "tools/StaticAnalysis/Exceptions/${ModuleName}/BreakingChangeIssues.csv"
> > "AssemblyFileName","ClassName","Target","Severity","ProblemId","Description","Remediation"
> > "Microsoft.Azure.PowerShell.Cmdlets.Network.dll","Microsoft.Azure.Commands.Network.NewAzFirewallPolicyNatRuleCommand","New-AzFirewallPolicyNatRule","0","1050","The parameter set '__AllParameterSets' for cmdlet 'New-AzFirewallPolicyNatRule' has been removed.","Add parameter set '__AllParameterSets' back to cmdlet 'New-AzFirewallPolicyNatRule'."
> > 
> > remove the unnecessary breaking change notifications
> > ```
> 
> @aim-for-better Please refer to my comments

Updated the file per comment

**Review by msJinLei — APPROVED** [review]

**Review by msJinLei — APPROVED** [review]

**Comment by msJinLei** [comment]
@aim-for-better Could provide the migration guide of HDInsight for this major release?

Please fill in the following table

|cmdlet|breaking change|old usage|new usage|
|-|-|-|-|

**Comment by aim-for-better** [comment]
> @aim-for-better Could you provide the migration guide of HDInsight for this major release?
> 
> Please fill in the following table
> 
> cmdlet	breaking change	old usage	new usage
> Thanks

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:///C:/Users/t-zhzho/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

<link rel=themeData
href="file:///C:/Users/t-zhzho/AppData/Local/Temp/msohtmlclip1/01/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:///C:/Users/t-zhzho/AppData/Local/Temp/msohtmlclip1/01/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=EN-US style='tab-interval:.5in;word-wrap:break-word'>




cmdlet | breaking change | old usage | new usage
-- | -- | -- | --
New-AzHDInsightCluster | Removed the   parameters “-RdpAccessExpiry” and “-RdpCredential” which   have been marked deprecated for a very long time | Customer didn’t   use the two parameters for a very long time.   There is not   old usage. | Don’t use the   two parameters any more.




</body>

</html>

**Pull request merged**